{"name": "Курьер не может войти с неверным паролем", "status": "passed", "steps": [{"name": "Залогинивание курьера (POST)", "status": "passed", "parameters": [{"name": "data", "value": "'{\"login\": \"\\u0411\\u043e\\u0431 \\u0414\\u0438\\u043b\\u0430\\u043d\", \"password\": \"09876\"}'"}, {"name": "status", "value": "404"}], "start": 1717328685473, "stop": 1717328685585}], "start": 1717328685473, "stop": 1717328685585, "uuid": "a8d32db7-ff08-4651-9961-7173bee36ae8", "historyId": "2f97d1bf5b6beaa54b11a72093030306", "testCaseId": "2f97d1bf5b6beaa54b11a72093030306", "fullName": "tests.test_login_courier.TestLoginCourier#test_login", "labels": [{"name": "feature", "value": "Проверка авторизации курьеров"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_login_courier"}, {"name": "subSuite", "value": "TestLoginCourier"}, {"name": "host", "value": "192.168.1.3"}, {"name": "thread", "value": "85775-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_login_courier"}]}